tests/files/dbc/BU_BO_REL_.dbc

```diff
@@ -47,7 +47,6 @@ BA_DEF_ BO_  "GenMsgCycleTime" INT 0 65535;
 BA_DEF_DEF_  "GenMsgCycleTime" 0;
 BA_DEF_REL_ BU_BO_REL_  "MsgProject" ENUM  "A","B","C";
 BA_DEF_DEF_REL_ "MsgProject" "A";
-BA_REL_ "MsgProject" BU_BO_REL_ COM 1 2;

```

Comment:
What is the reason for removal?

Reply:
This is not valid line. I checked with vector CANdb++. Since updated code handles this type of attributes, I need to remove it or update it with correct syntax

Reply:
Move the line BA_DEF_DEF_  "GenMsgCycleTime" 0; 
to come after BA_DEF_REL_ BU_BO_REL_  "MsgProject" ENUM  "A","B","C";
keep BA_REL_ "MsgProject" BU_BO_REL_ COM 1 2;
And file opens correct in Candb++.
Candb++ is sensitive for order of DEF_ and DEF_DEF_ s in dbc file.

Reply:
> @jak72 Yes, that is true. @malneni can you reorder to fix and use your CANdb++ to check that it loads properly?
I have handled Node-mapped signals but apparently there is Node-mapped Messages also. I have not handled this so I get test fail in BU_BO_REL_.dbc. I need more time to complete second feature. 
So, Can you consider these 2 as separate feature? & two separate pull request?